arangod/Network/Methods.cpp

```diff
@@ -234,6 +234,13 @@ class RequestsState final : public std::enable_shared_from_this<RequestsState> {
 
   // scheduler requests that are due
   void startRequest() {
+    if (!_pool) {
```

Comment:
```suggestion
    if (ADB_UNLIKELY(!_pool)) {
```

I think in reality, we will never loose the pool.

Reply:
But feel free to not include this, the code you have is correct, this is just some very minor perf optimization which will not be measurable. (as opposed to the network we are going to use)